x/genutil/client/cli/collect.go

```diff
@@ -13,12 +13,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/server"
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	"github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
```

Comment:
This defeats the point. You've now introduced a dependency on this module.

Reply:
Thanks for the re-review and apologizes for the delay in responding :bow: 

This change just maintains backwards compatibility, based on this discussion => https://github.com/cosmos/cosmos-sdk/pull/7279#discussion_r487128039 && this PR https://github.com/cosmos/cosmos-sdk/pull/7316